docs/try/tutorials/geomodel/geomodel-basics.md

```diff
@@ -1,22 +1,76 @@
 
-## How to build and use GeoModel
-### GeoModelBuilder
+## Basic functionalities on GeoModel
+The most efficient way to build a GeoModel is to import its geometry and topology from a
+geomodeler. This tutorial present a simple code to load manipulate and export `GeoModel`.
+
+### GeoModel I/O
+
+GeoModel stores the geometry and the topology of a geological model. It can be imported from boundary 
+representation or volumetric representation file format.
+RINGMesh uses an extensible design to handle various file formats.
+The file extention is the key for selecting the suitable loader/saver. 
+The Input/Output file formats currently supported by RINGMesh are listed [here](/features/file_formats).
+
+A GeoModel can be loaded and saved as follow:
+
+	 // GeoModel instantiation in 3D
+	 GeoModel3D geomodel ;
+	
+	 // LOADING
+	 //path to the file to load
+	 //the file extention is the factory key.
+	 std::string input_file_name = "path/to/your_input_model.key";
+	 //function to load a geomodel from input_file_name
+	 geomodel_load( geomodel, input_file_name ) ;
+	 
+	 //SAVING
+	 //path to the file to write
+	 //the file extention is the factory key.
+	 std::string output_file_name = "path/to/your_output_model.key"; 
+	 geomodel_save(geomodel, output_file_name);
+
+When a GeoModel is loaded, its validity is automatically checked and resulting information 
+are displayed in the consol terminal.
```

Comment:
Not sure it is a good idea to write were the display is. First, console and terminal are two different things: https://fossbytes.com/difference-between-shell-console-terminal/. Second, if you load in the viewer, you have another type of display. So why not just leave displayed? Where ? It depends...